modules/acquia_cms_common/tests/src/ExistingSite/ContentTypeListTestBase.php

```diff
@@ -143,7 +142,7 @@ protected function setUp(): void {
       'field_' . $this->nodeType . '_type' => $types[2],
       'created' => $time++,
     ]);
-
+    \Drupal::getContainer()->get('search_api.post_request_indexing')->destruct();
```

Comment:
Can we use `$this->container->get('search_api.post_request_indexing')->destruct();`

Reply:
Same as below